### DISCUSSION TITLE
若耶溪上 - 中国古诗词闲读

### DISCUSSION THREAD

**Discussion by guttasingapore** [RoadSideUniversity/RSU#1] [Announcements]
目的：与若耶溪相关引申的中国古诗词闲聊式漫读，每周一次，半小时左右在CH读一篇短诗，希望帮助各位包括我自己，了解历史，依托文化，漫谈情怀， 慰藉生活。

https://www.clubhouse.com/invite/RXpcV4KnBDegQGYYyjE4BrRB86O4HVojoBq:6SuUE6pwIryF5_wmDlfsjxfbk-p4Kdswca6brhjCnLA 

若耶溪上 - 中国古诗词闲读是一个CH上开放式的录音房，不谈时政，只谈诗词歌赋历史文化情怀生活。

关于若耶溪
http://www.shaoxing.com.cn/xinwen/p/2811412.html

第一读： 2024.1.23  千年若耶溪 - 开读《入若耶溪》南朝 王籍
https://baike.baidu.com/item/%E5%85%A5%E8%8B%A5%E8%80%B6%E6%BA%AA/3603331
CH录音https://www.clubhouse.com/room/P9gkVzRo?utm_medium=ch_room_pxr&utm_campaign=lpYfcJN9MC3YSlNJ-t3K_A-1082896

**Comment by Lewisrxliu** [comment]
GPT与唐诗的千年呼应，一点不违和

入若耶溪
王籍〔南朝〕
艅艎何泛泛，空水共悠悠。
阴霞生远岫，阳景逐回流。
蝉噪林逾静，鸟鸣山更幽。
此地动归念，长年悲倦游。

无题
必应GPT4， 2024.1.23
若耶溪边春色浓，
桃花红艳映清风。
游人不觉流年逝，
只愁芳草暮成空。

**Comment by guttasingapore** [comment]
第2读： 2024.1.30 《采莲曲》
唐，李白
若耶溪旁采蓮女，笑隔荷花共人語。 
日照新妝水底明，風飄香袂空中舉。 
岸上誰家游冶郎，三三五五映垂楊。 
紫騮嘶入落花去，見此踟躕空斷腸。

**Reply by Lewisrxliu** [reply]
https://www.clubhouse.com/room/7VBCZ8ydVjtG6Z675:AJc0gZOwFTp_GZ7_JNMNz8LFPnPsHQogqyLgJ5blZVk?utm_medium=ch_instant_invite&utm_campaign=lpYfcJN9MC3YSlNJ-t3K_A-1092598

**Reply by Lewisrxliu** [reply]
笑隔荷花共人语- 宋.秦观
若耶溪边天气秋。采莲女儿溪岸头。笑隔荷花共人语，烟波渺渺荡轻舟。数声水调红娇晚。棹转舟回笑人远。肠断谁家游冶郎，尽日踟蹰临柳岸。

**Reply by Lewisrxliu** [reply]
笑隔荷花共人语- 宋.秦观
若耶溪边天气秋。采莲女儿溪岸头。笑隔荷花共人语，烟波渺渺荡轻舟。数声水调红娇晚。棹转舟回笑人远。肠断谁家游冶郎，尽日踟蹰临柳岸。

**Comment by guttasingapore** [comment]
2.6.2024 《入若耶溪》唐朝 崔颢
轻舟去何疾，已到云林境。起坐鱼鸟间，动摇山水影。
岩中响自答，溪里言弥静。事事令人幽，停桡向余景。

**Comment by Lewisrxliu** [comment]
2024.2.16 王维题友人所居《皇甫岳云溪杂题》五首
《鸟鸣涧》
[唐]王维
人闲桂花落，夜静春山空。
月出惊山鸟，时鸣春涧中。

**Reply by Lewisrxliu** [reply]
禅者王维 https://www.jianshu.com/p/837602cc0d7c
温柔地走进唐诗中的那些良夜- 三书 https://www.163.com/dy/article/G77HUIGO051284DV.html
https://zh.wikipedia.org/zh-hans/王维
https://zh.wikipedia.org/wiki/王缙

**Comment by Lewisrxliu** [comment]
2024.5.28 若耶溪与孟浩然
https://www.clubhouse.com/room/PY7NVo2d?utm_medium=ch_room_pxr&utm_campaign=lpYfcJN9MC3YSlNJ-t3K_A-1264021

孟浩然《游云门寺寄越州府包户曹徐起居》：“我行适诸越，梦寐怀所欢。久负独往愿，今来恣游盘。台岭践磴石，耶溪溯林湍”

孟浩然《耶溪泛舟》：
落景余清辉，轻桡弄溪渚。
泓澄爱水物，临泛何容与。
白首垂钓翁，新妆浣纱女。
相看似相识，脉脉不得语。


《赠孟浩然》 李白
吾爱孟夫子，风流天下闻。红颜弃轩冕，白首卧松云。醉月频中圣，迷花不事君。高山安可仰，徒此揖清芬

《解闷十二首(其六)》 杜甫 “复忆襄阳孟浩然，清诗句句尽堪传。即今耆旧无新语，漫钓槎头缩颈鳊。

转喉触讳:玄宗召李入对，因从容说及孟浩然。李奏曰：“臣故人也，见（现）在臣私第。”上急召赐对，俾口进佳句。孟浩然诵诗曰：“北阙休上书，南山归敝庐。不才明主弃，多病故人疏。”上意不悦，乃曰：“未曾见浩然进书，朝廷退黜。何不云：气蒸云梦泽，波动岳阳城？缘是不降恩，终于布衣而已

**Reply by Lewisrxliu** [reply]
《黄鹤楼送孟浩然之广陵》 李白
故人西辞黄鹤楼，烟花三月下扬州。
孤帆远影碧空尽，唯见长江天际流

孟浩然最经典的几首唐诗
1，春晓
春眠不觉晓，处处闻啼鸟。
夜来风雨声，花落知多少。

2，过故人庄
故人具鸡黍，邀我至田家。
绿树村边合，青山郭外斜。
开轩面场圃，把酒话桑麻。
待到重阳日，还来就菊花。

3，宿建德江
移舟泊烟渚，日暮客愁新。
野旷天低树，江清月近人。

4，与诸子登岘山
人事有代谢，往来成古今。
江山留胜迹，我辈复登临。
水落鱼梁浅，天寒梦泽深。
羊公碑尚在，读罢泪沾襟。

5，临洞庭上张丞相
八月湖水平，涵虚混太清。
气蒸云梦泽，波撼岳阳城。
欲济无舟楫，端居耻圣明。
坐观垂钓者，徒有羡鱼情。

6，洛中访袁拾遗不遇
洛阳访才子，江岭作流人。
闻说梅花早，何如北地春。

7. 《岁暮归南山》
北阙休上书，南山归敝庐。
不才明主弃，多病故人疏。
白发催年老，青阳逼岁除。
永怀愁不寐，松月夜窗虚。

8.《早寒江上有怀》
木落雁南度，北风江上寒。
我家襄水曲，遥隔楚云端。
乡泪客中尽，孤帆天际看。
迷津欲有问，平海夕漫漫。

9.《夏日南亭怀辛大》
山光忽西落，池月渐东上。
散发乘夕凉，开轩卧闲敞。
荷风送香气，竹露滴清响。
欲取鸣琴弹，恨无知音赏。
感此怀故人，中宵劳梦想。

10《夜归鹿门山歌》
山寺钟鸣昼已昏，渔梁渡头争渡喧。
人随沙岸向江村，余亦乘舟归鹿门。
鹿门月照开烟树，忽到庞公栖隐处。
岩扉松径长寂寥，惟有幽人自来去。

https://baike.baidu.com/item/孟浩然/28158
https://zh.wikipedia.org/zh-hans/堕泪碑

**Comment by Lewisrxliu** [comment]
2024.8.20 第六读：千年若耶溪之刘长卿
上巳日越中与鲍侍郎泛舟耶溪
兰桡缦转傍汀沙，应接云峰到若耶。旧浦满来移渡口，
垂杨深处有人家。永和春色千年在，曲水乡心万里赊。
君见渔船时借问，前洲几路入烟花。

送李校书适越谒杜中丞
江风处处尽，旦暮水空波。
摇落行人去，云山向越多。
陈蕃悬榻待，谢客枉帆过。
相见耶溪路，逶迤入薜萝。

http://www.shiren.org/xlib/lingshidao/gushi/liuchangqing.htm
http://m.shangshiwen.com/shiren/104_1.html
https://zh.wikipedia.org/wiki/刘长卿

逢雪宿芙蓉山
日暮苍山远，天寒白屋贫。
柴门闻犬吠，风雪夜归人。

《寻南溪常山道人隐居》
一路经行处，莓苔见履痕。
白云依静渚，芳草闭闲门。
过雨看松色，随山到水源。
溪花与禅意，相对亦忘言。

听弹琴
泠泠七丝上，静听松风寒。
古调虽自爱，今人多不弹。

送灵澈上人
苍苍竹林寺，杳杳钟声晚。
荷笠带斜阳，青山独归远。

长沙过贾谊宅
三年谪宦此栖迟，万古惟留楚客悲。
秋草独寻人去后，寒林空见日斜时。
汉文有道恩犹薄，湘水无情吊岂知？
寂寂江山摇落处，怜君何事到天涯！

饯别王十一南游
望君烟水阔，挥手泪沾巾。
飞鸟没何处，青山空向人。
长江一帆远，落日五湖春。
谁见汀洲上，相思愁白苹。

重送裴郎中贬吉州
猿啼客散暮江头，人自伤心水自流。
同作逐臣君更远，青山万里一孤舟。

新年作
乡心新岁切，天畔独潸然。
老至居人下，春归在客先。
岭猿同旦暮，江柳共风烟。
已似长沙傅，从今又几年。

弹琴
冷冷七弦上，静听松风寒。
古调虽自爱，今人多不弹。

送上人
孤云将野鹤，岂向人间住。
莫买沃洲山，时人已知处。

《别严士元》
春风倚棹阖闾城，水国春寒阴复晴。
细雨湿衣看不见，闲花落地听无声。
日斜江上孤帆影，草绿湖南万里情。
东道若逢相识问，青袍今已误儒生。

余干旅舍
摇落暮天迥，青枫霜叶稀。
孤城向水闭，独鸟背人飞。
渡口月初上，邻家渔未归。
乡心正欲绝，何处捣寒衣。

**Comment by Lewisrxliu** [comment]
2024.9.11 第七读：千年若耶之其他唐代诗人（未完可待续）
这么多唐人唐诗我们可能都没有听说过，盘点片刻居然还有超过二十多位在若耶诗川留下了属于他们的时光痕迹，在继续聊关于若耶溪的宋诗宋词之前，让我们再一次理解唐代的诗文审美吧。

丘为《泛若耶谿》“一川草长绿，四时那得辨”
丘为《泛若耶谿》“结庐若耶里，左右若耶水。......溪中水流急，渡口水流宽”；
https://zh.wikipedia.org/zh-sg/丘为
https://www.gushiwen.cn/Author_39266de739.aspx

皎然《泛耶溪》“芳草行无尽，清源去不穷。”
皎然《泛耶溪》“春生若耶水，雨后漫流通”，
https://zh.wikipedia.org/wiki/皎然

綦毋潜《春泛若耶溪》“好风吹行舟，花路入溪口”；

朱庆馀《过耶溪》“清溪缭绕出无穷，两岸桃花正好风”；
朱庆馀《过耶溪》“却是扁舟堪入处，鸳鸯飞起急流中”；

严维《奉和独孤中丞游云门寺》“绝壑开花界，耶溪极上源”
严维《奉和独孤中丞游云门寺》：“苍翠新秋色，莓苔积雨痕。上方看度鸟，后夜听吟猿。”
严维等联句《云门寺小溪茶宴怀院中诸公》“猨饮无人处，琴听浅溜边”，《一公新泉》“独映孤松色，殊分众鸟喧”，

罗邺《宿云门寺》“入松穿竹路难分，藉地连岩总是云”

储光羲《酬綦毋校书梦游耶溪见赠之作》“往往揽垂葛，出舟望前林。山人松下饮，钓客芦中吟”
储光羲《酬綦毋校书梦游耶溪见赠之作》“春看湖口漫，夜入回塘深”；

顾非熊《入云门五云溪上作》“沿山寺寺花树，枕水家家竹林”
顾非熊《入云门五云溪上作》：“忽闻风动莲叶，起见波间月沉。”
顾非熊《入云门五云溪上作》“鸳鸯昼飞溪静，鸜鹆夜转村深”；

常建《白湖寺后溪宿云门》“落日山水清，乱流鸣淙淙。旧蒲雨抽节，新花水对窗”；

赵嘏《游云门》“五云溪影里，万虑淡凉天”；

于武陵《泛若耶溪宿云门》“溪船泛数里,渐觉灭炎辉。......深犹见白石,凉好换生衣。未得多诗句,终须隔宿归”

项斯《宿云门寺》“山果经霜多自落，水萤穿竹不停飞”

权德舆《人日送房二十六侍御归越》：“驿骑归时骢马蹄，莲花府映若耶溪”

刘得仁《送越客归》：“到家冬即是，荷尽若耶溪。”

萧翼《宿云门东客院》：
路入山西更向西，雨和春雪旋成泥。风吹叠巘云头散。月照平湖沿雁影低。拄杖负书寻远寺，倩童牵鹿渡深溪。今朝独宿岩东院，唯听猿吟与鸟啼。

灵澈《云门寺》“天寒猛虎叫岩月，松下无人空有雪”，
灵澈《云门寺》“湖边归鹤唳寥泬，僧房半依秦峰缺”

方干《再游云门》“僧老终无出山意，岩猿涧鸟莫相猜”
方干《思越中旧游寄友》“断臂青猿啼玉笥，成群白鸟下耶溪”；

许浑《泛五云溪》“鱼倾荷叶露，蝉噪柳林风”；

于武陵《泛若耶溪宿云门》“暎水花连影,逢人鸟背飞”；

宋之问《忆云门》“树闲烟不破，溪静鹭忘飞”。

李绅《若耶溪》
岚光花影绕山阴，山转花稀到碧浔。倾国美人妖艳远，凿山良冶铸炉深。凌波莫惜临妆面，莹锷当期出匣心。应是蛟龙长不去，若耶秋水尚沈沈。

唐，杜甫《奉先刘少府新画山水障歌》:
“ 若耶溪，云门寺，吾独胡为在泥滓？青鞋布袜从此始。”

沈约《宋书》曰：“会稽带海傍湖，良畴数千万顷，膏腴上地，亩直一金，鄂、杜之间，不能比也。”

袁宏道《山阴道》云：“六朝以上人,不闻西湖好。平生王献之,酷爱山阴道。彼此俱清奇,输他得名早。”
